holidays/countries/guyana.py

```diff
@@ -0,0 +1,263 @@
+    SUN_TO_NEXT_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+)
+
+
+class Guyana(
+    ObservedHolidayBase,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """Guyana holidays.
+
+    References:
+        * [Public Holidays Amendment Act 1967](https://web.archive.org/web/20241125073357/https://parliament.gov.gy/documents/bills/11264-bill_6_of_1967_public_holidays.pdf)
+        * [Public Holidays Amendment Act 1969](https://web.archive.org/web/20250424211718/https://parliament.gov.gy/documents/bills/11891-bill_26_of_1969_1.pdf)
+        * [Public Holidays Act Consolidated as of 2012](https://mola.gov.gy/laws/Volume%206%20Cap.%2018.01%20-%2023.011696968337.pdf)
+    """
+
+    country = "GY"
+    default_language = "en_GY"
+    # %s (estimated).
+    estimated_label = tr("%s (estimated)")
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observed, estimated)")
+    start_year = 1968
+    supported_languages = ("en_GY", "en_US")
+
+    def __init__(self, *args, islamic_show_estimated: bool = True, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=GuyanaHinduHolidays)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=GuyanaIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        StaticHolidays.__init__(self, cls=GuyanaStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(
+            self._add_new_years_day(tr("New Year's Day")), rule=SAT_SUN_TO_NEXT_WORKDAY
+        )
+
+        if self._year <= 1969:
+            # Independence Day.
+            self._add_holiday_may_26(tr("Independence Day"))
+
+        if self._year >= 1970:
+            # Republic Day.
+            self._add_holiday_feb_23(tr("Republic Day"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Easter Monday"))
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Labour Day")))
+
+        # Commonwealth Day.
+        self._add_holiday_1st_mon_of_aug(tr("Commonwealth Day"))
+
+        # Christmas Day.
+        self._add_observed(
+            self._add_christmas_day(tr("Christmas Day")), rule=SAT_SUN_TO_NEXT_MON_TUE
+        )
+
+        # Day after Christmas.
+        self._add_observed(
+            self._add_christmas_day_two(tr("Day after Christmas")), rule=SAT_SUN_TO_NEXT_MON_TUE
+        )
+
+        # Phagwah Day.
```

Comment:
```suggestion
        # Holi.
```
More standard name for this holiday (although there are not so many examples of it in other countries).